### PR TITLE
#133 Issue: Footer responsive and icons issue was fixed.

### LINKED ISSUE #133: 🐛 Bug Report: Footer Responsive issue
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

On small size devices, the social icons are missing and email is also not perfectly align.

image

### Expected Behavior

add all other social medial icons and fix the email alignment and typo.

image

checkout this design to inspire and create a component https://www.figma.com/file/ZTnjjETRtg8HNV0zufJajF/GDSC?node-id=0%3A1&t=684eNs6xDEXUA2GK-1

### Steps To Reproduce

_No response_

### Anything else?

_No response_

### What browsers are you seeing the problem on?

_No response_

### PR DESCRIPTION
**Pull request by omernadeem1070**
**Notes for Reviewers**

This PR fixes #133 

Footer responsive issue was fixes as well as icons disappearing issue was fixed.

Screenshots for the change:
image

### CODE REVIEW

File: src/Pages/Footer/Footer.css [hunk 1]
```diff
@@ -178,6 +187,24 @@
...
+    }
+
+    .grid_container__box4{
+        display: block
+    }
+    .grid_container__box4 > div{
+        display: flex;
+        justify-content: center;
+     }
+     .grid_container__box4 > div > svg{
```
Comment (anshitmishra) [inline src/Pages/Footer/Footer.css:201]: here if we change margin-right:40px;
 
to margin:0xp 20px; that changes the SVG icon in center now svg icons are in right size

**Comment by anshitmishra** [comment]
@omernadeem1070 great work just one more suggestion make email and location clickable so when the user clicks on email it will redirect to email and then click on a location it will redirect to a location in a new tab

File: src/Pages/Footer/Footer.css (hunk 1 above)
Comment (Yashsharma1911) [reply src/Pages/Footer/Footer.css:201]: @omernadeem1070

**Comment by ArunPatel02** [comment]
@omernadeem1070 i am getting uneven distribution of space between icons. can anyone confirm this if they are getting the same. @anshitmishra @Yashsharma1911 @martinyis @rodrigoluchina  
image

**Comment by ArunPatel02** [comment]
> @omernadeem1070 i am getting uneven distribution of space between icons. can anyone confirm this if they are getting the same. @anshitmishra @Yashsharma1911 @martinyis @rodrigoluchina  
> image
> 

@omernadeem1070 any updates?

File: src/Pages/Footer/Footer.css (hunk 1 above)
Comment (omernadeem1070) [reply src/Pages/Footer/Footer.css:201]: I'll check

**Comment by omernadeem1070** [comment]
Sure i'll check.

**Review by kamalchakma1 — APPROVED** [review]

**Pull request closed without merging**